src/packages/user/current-user/current-user-header-app.element.ts

```diff
@@ -45,14 +51,70 @@ export class UmbCurrentUserHeaderAppElement extends UmbLitElement {
 		this.#modalManagerContext?.open(UMB_CURRENT_USER_MODAL);
 	}
 
+	async #getAppContext() {
+		// TODO: remove this when we get absolute urls from the server
+		return this.consumeContext(UMB_APP_CONTEXT, (instance) => {}).asPromise();
+	}
+
+	#setUserAvatarUrls = async (user: UmbCurrentUser | undefined) => {
+		if (!user || !user.avatarUrls || user.avatarUrls.length === 0) {
+			this._userAvatarUrls = [];
+			return;
+		}
+
+		// TODO: remove this when we get absolute urls from the server
+		// TODO: temp hack because we can't prefix local urls with the server url.
+		// these are preview urls for newly uploaded avatars
+		const serverUrl = (await this.#getAppContext()).getServerUrl();
+		if (!serverUrl) return;
+
+		// TODO: hack to only use size 3 and 4 from the array. The server should only return 1 url.
+		const isRelativeUrl = user.avatarUrls[0].startsWith('/');
+		const avatarScale1 = user.avatarUrls?.[0];
+		const avatarScale2 = user.avatarUrls?.[1];
+		const avatarScale3 = user.avatarUrls?.[2];
+
+		this._userAvatarUrls = [
+			{
+				scale: '1x',
+				url: isRelativeUrl ? serverUrl + avatarScale1 : avatarScale1,
+			},
+			{
+				scale: '2x',
+				url: isRelativeUrl ? serverUrl + avatarScale2 : avatarScale2,
+			},
+			{
+				scale: '3x',
+				url: isRelativeUrl ? serverUrl + avatarScale3 : avatarScale3,
+			},
```

Comment:
@madsrasmussen maybe rename `scale` to `density` if it is directly related to the `srcset` attribute as in current backoffice. Instead of 1x, 2x, 3x (DPR - device pixels ratio) it could also have used `w` based on viewport size, e.g. 600w, 1000w etc.

Reply:
Hi Bjarne, good feedback,  I think renaming scale to density sounds good. I would recommend making a PR with that change, then I will happily approve it :-)

This one is getting merged for now.